public/js/src/module/gui.js

```diff
@@ -184,56 +180,18 @@ function swapTheme(formParts) {
     });
 }
 
-feedbackBar = {
-    /**
-     * Shows an unobtrusive feedback bar to the user.
-     *
-     * @param { string } message - message to show
-     * @param {number=} duration - duration in seconds for the message to show
-     */
-    show(message, duration) {
-        let $msg;
-        const $fbBar = $('#feedback-bar');
-
-        duration = duration ? duration * 1000 : 10 * 1000;
-
-        // max 2 messages displayed
-        $fbBar.addClass('feedback-bar--show').find('p').eq(1).remove();
-
-        // if an already shown message isn't exactly the same
-        if ($fbBar.find('p').html() !== message) {
-            $msg = $('<p></p>').append(message);
-            $fbBar.prepend($msg);
-        }
-
-        // automatically remove feedback after a period
-        setTimeout(() => {
-            let siblings;
-            if (typeof $msg !== 'undefined') {
-                siblings = $msg.siblings('p').length;
-                $msg.remove();
-                if (siblings === 0) {
-                    feedbackBar.hide();
-                }
-            }
-        }, duration);
-    },
-    hide() {
-        $('#feedback-bar').removeClass('feedback-bar--show').find('p').remove();
-    },
-};
-
 /**
  * Select what type of unobtrusive feedback message to show to the user.
  *
  * @param { string }  message - message to show
  * @param {number=} duration - duration in seconds for the message to show
+ * @param {string=} heading - heading to show (on mobile only)
  */
-function feedback(message, duration) {
+function feedback(message, duration, heading = t('feedback.header')) {
```

Comment:
In enketo-webform.js there are 2 gui.feedback calls that provide a 3rd parameter. That parameter seemed useful but wasn't used.